community-operators [R] community-kubevirt-hyperconverged (0.0.1 0.0.2 0.0.3 1.0.0 1.1.0)

orenc1 opened the pull request:
HCO versions 1.4.0 and 1.4.1 we published prematurely.
We would like to align it with our official release schedule.
Until then, users can consume latest HCO from master using the auto-updated index image of:
quay.io/kubevirt/hyperconverged-cluster-index:1.4.0-unstable

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Does your operator have [nested directory structure](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#create-a-bundle)?
* [ ] Have you selected the Project *Community Operator Submissions* in your PR on the right-hand menu bar?
* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#updating-your-existing-operator)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#updating-your-existing-operator)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#bundle-format) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Submit operators to both `upstream-community-operators` and `community-operators` at once
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-required-fields.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.svg)

orenc1 commented:
/cc @tiraboschi

mvalarh commented:
Please have a look: https://github.com/operator-framework/operator-registry/issues/630

orenc1 commented:
> Please have a look: [operator-framework/operator-registry#630](https://github.com/operator-framework/operator-registry/issues/630)

Thanks, Martin.
I removed the `defaultChannel` annotation from bundles that are not part of the `stable` channel, which we want to be the default channel.

mvalarh commented:
@orenc1 i removed check for default channel. So let's see now how it goes

orenc1 commented:
Thanks, @mvalarh.
it is now failing on:
`The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'operators.operatorframework.io.bundle.channel.default.v1'`
in task: `detect_format : Set operators variables from annotation file`

I think that change should be updated in other places as well

mvalarh commented:
@orenc1 it is fixed in pipeline. Please have a look for errors in new logs now.

orenc1 commented:
@mvalarh , thanks, I fixed the annotations.
Any idea why the openshift-ci test is failing?
I'm getting:
```
Temp index not found. Are your commits squashed? If so, please check logs https://github.com/operator-framework/community-operators/actions?query=workflow%3Aprepare-test-index
```
I rebased the PR and it is a single commit.
Also, I'm removing versions 1.4.0 and 1.4.1 - why I'm seeing "OP_VER=1.4.1" [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/operator-framework_community-operators/3683/pull-ci-operator-framework-community-operators-master-deploy-operator-on-openshift/1387401315604238336#1:build-log.txt%3A179)?

Thanks!

mvalarh commented:
/test all

mvalarh commented:
Let me retest it

J0zi commented:
/test all

J0zi commented:
working on pipeline fix

J0zi commented:
/test all

J0zi commented:
Merging as test is not needed upon deletion. Deployments are green.
